### PR TITLE
Logging mix completion for client

### PR DESCRIPTION
**Pull request by psinha25**

### CODE REVIEW

File: src/scheduler.py [hunk 1]
```diff
@@ -199,6 +200,7 @@ def UpdateClusterState(self, request, context):
             echo_dict = {"load": request.load, "run_id": request.run_id}
             echo_dict_string = json.dumps(echo_dict)
             echo_cmd = "echo \'" + echo_dict_string + f"\' > /tmp/tiebreaker-completed-mixes.txt"
+            echo_cmd = f"echo {request.run_id} > /tmp/tiebreaker-completed-mixes.txt"
```
Comment (aravindhank11) [inline src/scheduler.py]: Shouldn't this be: 
```
echo_cmd = f"echo {request.run_id} >> /tmp/tiebreaker-completed-mixes.txt"
```

File: src/scheduler.py [hunk 2]
```diff
@@ -199,6 +200,7 @@ def UpdateClusterState(self, request, context):
             echo_dict = {"load": request.load, "run_id": request.run_id}
             echo_dict_string = json.dumps(echo_dict)
             echo_cmd = "echo \'" + echo_dict_string + f"\' > /tmp/tiebreaker-completed-mixes.txt"
```
Comment (aravindhank11) [inline src/scheduler.py]: Should this be removed?

File: src/scheduler.py (hunk 1 above)
Comment (psinha25) [reply src/scheduler.py]: Yes my bad. Fixed

File: src/scheduler.py (hunk 2 above)
Comment (psinha25) [reply src/scheduler.py]: Removed also

**Pull request merged**